pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/EgenerklaeringPleieOgOmsorgsarbeid.kt

```diff
@@ -102,7 +102,7 @@ val egenerklaeringPleieOgOmsorgsarbeid = createAttachment<LangBokmalNynorskEngli
         )
     )
 
-    repeat(4) { newline() }
```

Comment:
Fikser disse endringene at skjemaet går over to sider?

Reply:
Ja.